### PR TITLE
Minor code clean up.

### PR DESCRIPTION
**Pull request by alexbasson**
- Remove unused variables
- Change declaration of constants from var to let
- Remove unnecessary optionals and force-unwrapping

Signed-off-by: alexbasson <alexbasson@gmail.com>

### CODE REVIEW

File: Herald/Herald/Sensor/Location/MobilitySensor.swift [hunk 1]
```diff
@@ -19,9 +19,10 @@ protocol MobilitySensor : Sensor {
  Requires : Info.plist : Privacy - Location Always and When In Use Usage Description
  */
 class ConcreteMobilitySensor : NSObject, MobilitySensor, CLLocationManagerDelegate {
+    public static let minimumResolution: Distance = Distance(kCLLocationAccuracyThreeKilometers)
+
     private let logger = ConcreteSensorLogger(subsystem: "Sensor", category: "ConcreteMobilitySensor")
     /// Minimum mobility sensing resolution is 3km as defined by CoreLocation.
```
Comment (adamfowleruk) [inline Herald/Herald/Sensor/Location/MobilitySensor.swift:25]: This comment needs moving too.

File: Herald/Herald/Sensor/Motion/InertiaSensor.swift [hunk 2]
```diff
@@ -69,11 +44,36 @@ class ConcreteInertiaSensor : NSObject, InertiaSensor {
         logger.debug("start")
         motionManager.accelerometerUpdateInterval = updateInterval
         motionManager.stopAccelerometerUpdates()
-        motionManager.startAccelerometerUpdates(to: operationQueue, withHandler: accelerometerHandler)
+        motionManager.startAccelerometerUpdates(to: operationQueue, withHandler: handleAccelerometerUpdates)
```
Comment (adamfowleruk) [inline Herald/Herald/Sensor/Motion/InertiaSensor.swift:47]: Does this approach work with iOS 9.3+ or only more recent versions?

Comment (alexbasson) [reply Herald/Herald/Sensor/Motion/InertiaSensor.swift:47]: Functions are first-class data in Swift, so this _ought_ to work with all versions of Swift.

File: Herald/Herald/Sensor/Location/MobilitySensor.swift (hunk 1 above)
Comment (alexbasson) [reply Herald/Herald/Sensor/Location/MobilitySensor.swift:25]: Do'h! My bad. I'll fix that.

Comment (alexbasson) [reply Herald/Herald/Sensor/Location/MobilitySensor.swift]: Fixed

**Comment by adamfowleruk** [comment]
@alexbasson Please merge develop into this branch and push again so the CI tests run. Thanks!

**Pull request merged**